decidim-admin/spec/system/participatory_space_private_user_import_via_csv_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "import private users via csv with the possibility to deleting some", type: :system do
```

Comment:
I'd change this to be a bit more consistent with the rest of the other specs 
```suggestion
describe "Admin manages participatory space private users via csv import", type: :system do
```